Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
My first approach was to use official [java](https://hub.docker.com/_/java/) `java:8` but as mentioned in https://github.com/jhipster/jhipster-docker/pull/19, Julien prefers to use an Ubuntu image to be consistent with their [DevBox](https://github.com/jhipster/jhipster-devbox).